tests/sentry/workflow_engine/migration_helpers/test_migrate_alert_rule.py

```diff
@@ -166,61 +167,81 @@ def assert_alert_rule_trigger_migrated(alert_rule_trigger):
 
 
 def build_sentry_app_compare_blob(
-    sentry_app_config: list[dict[str, str]]
-) -> list[dict[str, str | None]]:
+    sentry_app_config: list[dict[str, Any]],
+) -> list[dict[str, Any | None]]:
     """
     Add the label to the config
     """
     return [{**config, "label": config.get("label", None)} for config in sentry_app_config]
 
 
-def assert_alert_rule_trigger_action_migrated(alert_rule_trigger_action, action_type):
-    aarta = ActionAlertRuleTriggerAction.objects.get(
-        alert_rule_trigger_action=alert_rule_trigger_action
-    )
-    action = Action.objects.get(id=aarta.action_id, type=action_type)
-    assert DataConditionGroupAction.objects.filter(
-        action_id=action.id,
-    ).exists()
+def assert_sentry_app_action_migrated(
```

Comment:
do we need to test all of these conditions every time? could we setup tests in a way that we can test each of these attributes in isolation?

The thought behind testing in isolation is that if there's an issue with the `target_identifier` we can catch that issue _and_ continue to test that the `action.data` is correct too. By limiting the scope of those tests we can quickly identify any issues that come up in the future too.